modules/nw-pod-network-connectivity-implementation.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+//
+
+[id="nw-pod-network-connectivity-implementation_{context}"]
+= Implementation of connection health checks
+
+The connectivity check controller orchestrates connectivity verification checks in a {product-title} cluster. The results for the connection tests are stored in `PodNetworkConnectivity` objects in the `openshift-network-diagnostics` namespace. Connection tests are performed every minute in parallel.
+
+The Cluster Network Operator (CNO) deploys several resources to the cluster to send and receive connectivity health checks:
```

Comment:
Probably a dumb question: _connectivity verification checks_ and _connectivity health checks_ are the same thing??

Reply:
@mburke5678, it's not. I should probably use the same term throughout to avoid any confusion.